cmd/gmnhg/main.go

```diff
@@ -403,6 +439,44 @@ func main() {
 		panic(err)
 	}
 
+	// render RSS/Atom feeds
+	if tmpl, hasTmpl := templates["_default/rss"]; hasTmpl {
+		defaultRssTemplate = tmpl
+	}
+	for dirname, posts := range topLevelPosts {
+		// do not render RSS for leaf paths
+		if hasSubPath(leafIndexPaths, path.Join(contentBase, dirname)+"/") {
+			continue
+		}
+		tmpl, hasTmpl := templates["top"+dirname+"_rss"]
```

Comment:
I could possibly see this interfering with actual directory names ending with `_rss`. This is unlikely, but will probably look weird to someone starting an actual blog series on RSS (theoretic use of a top-level dir named `posts_rss`). Maybe we could introduce a new template directory for this?

Reply:
I changed it to gmnhg/rss, also added some documentation at the top of cmd/gmnhg/main.go. I think the package as a whole could use some more detailed documentation (with examples) in a doc.go file, but this should work as minimally adequate documentation.